packages/ra-ui-materialui/src/detail/Edit.js

```diff
@@ -92,6 +92,7 @@ export const EditView = ({
                             hasShow,
                             hasList,
                             resource,
+                            ...actions.props
```

Comment:
In here `basePath`, `resource`, `hasList`, `hasShow`, `data` cannot be customized without it.